packages/evm/contracts/registry/GuardiansRegistry.sol

```diff
@@ -407,130 +407,132 @@ contract JurorsRegistry is ControlledRecoverable, IJurorsRegistry, ERC900, Appro
-    * @return Total amount of active juror tokens
+    * @dev Tell the total amount of active guardian tokens
+    * @return Total amount of active guardian tokens
     */
     function totalActiveBalance() external view returns (uint256) {
         return tree.getTotal();
     }
 
     /**
-    * @dev Tell the total amount of active juror tokens at the given term id
+    * @dev Tell the total amount of active guardian tokens at the given term id
     * @param _termId Term ID querying the total active balance for
-    * @return Total amount of active juror tokens at the given term id
+    * @return Total amount of active guardian tokens at the given term id
     */
     function totalActiveBalanceAt(uint64 _termId) external view returns (uint256) {
         return _totalActiveBalanceAt(_termId);
     }
 
     /**
-    * @dev ERC900 - Tell the total amount of tokens of juror. This includes the active balance, the available
+    * @dev ERC900 - Tell the total amount of tokens of guardian. This includes the active balance, the available
     *      balances, and the pending balance for deactivation. Note that we don't have to include the locked
     *      balances since these represent the amount of active tokens that are locked for drafts, i.e. these
-    *      are included in the active balance of the juror.
-    * @param _juror Address of the juror querying the total amount of tokens staked of
-    * @return Total amount of tokens of a juror
+    *      are included in the active balance of the guardian.
+    * @param _guardian Address of the guardian querying the total amount of tokens staked of
+    * @return Total amount of tokens of a guardian
     */
-    function totalStakedFor(address _juror) external view returns (uint256) {
-        return _totalStakedFor(_juror);
+    function totalStakedFor(address _guardian) external view returns (uint256) {
+        return _totalStakedFor(_guardian);
     }
 
     /**
-    * @dev Tell the balance information of a juror
-    * @param _juror Address of the juror querying the balance information of
-    * @return active Amount of active tokens of a juror
-    * @return available Amount of available tokens of a juror
+    * @dev Tell the balance information of a guardian
+    * @param _guardian Address of the guardian querying the balance information of
+    * @return active Amount of active tokens of a guardian
+    * @return available Amount of available tokens of a guardian
     * @return locked Amount of active tokens that are locked due to ongoing disputes
     * @return pendingDeactivation Amount of active tokens that were requested for deactivation
     */
-    function balanceOf(address _juror) external view returns (uint256 active, uint256 available, uint256 locked, uint256 pendingDeactivation) {
-        return _balanceOf(_juror);
+    function balanceOf(address _guardian) external view
+        returns (uint256 active, uint256 available, uint256 locked, uint256 pendingDeactivation)
```

Comment:
Nitpick here, might want to double check how the solidity style guide expects these to be formatted

Reply:
I created an issue to review these at the end, now it's gonna be a mess with all the opened PRs